js/modules/loki_public_chat_api.js

```diff
@@ -203,18 +204,21 @@ class LokiPublicChannelAPI {
     this.serverAPI = serverAPI;
     this.channelId = channelId;
     this.baseChannelUrl = `channels/${this.channelId}`;
-    this.groupName = 'unknown';
     this.conversationId = conversationId;
+    this.conversation = ConversationController.get(conversationId);
```

Comment:
Will leave this for now and implement the fix discussed yesterday later

Reply:
which was? a check or revert? or just making sure we stop timers and deallocate channel API too?

Reply:
Ah just the discussion with Sach about grabbing this when we need it rather than storing a reference on the api object

Reply:
ok so a revert